backend/django/project/settings.py

```diff
@@ -75,8 +75,12 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'postgres',
```

Comment:
Consider creating a separate database user with permissions only to the tables needed for the application. Security best practice. :)